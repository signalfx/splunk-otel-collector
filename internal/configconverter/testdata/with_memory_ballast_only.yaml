receivers:
  otlp:
    protocols:
      grpc:
      http:

extensions:
  memory_ballast:
    size_mib: 120

service:
  extensions: [memory_ballast]
  pipelines:
    metrics:
      receivers: [otlp]