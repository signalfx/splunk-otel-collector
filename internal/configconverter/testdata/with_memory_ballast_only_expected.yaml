receivers:
  otlp:
    protocols:
      grpc:
      http:
service:
  extensions: []
  pipelines:
    metrics:
      receivers: [otlp]